src/utils/rfq_client.ts

```diff
@@ -98,4 +123,81 @@ export class RfqClient implements IRfqClient {
             };
         }
     }
+
+    /**
+     * Communicates to an RFQ Client to fetch available v2 prices
+     */
+    public async getV2PricesAsync(request: RfqtV2Request): Promise<RfqtV2Prices> {
+        // Short circuit if not rolled out
+        if (!RfqClient.isRolledOut(request)) {
+            return [];
+        }
+
+        try {
+            const response = await this._axiosInstance.post(`${this._rfqApiUrl}/internal/rfqt/v2/prices`, request, {
+                timeout: RFQT_REQUEST_MAX_RESPONSE_MS + 150,
+                headers: {
+                    '0x-chain-id': request.chainId,
```

Comment:
chain ID in the header is not necessary. We have it in the request body.

Reply:
will keep it for now, just because I want to keep the dream of forwarding the `0x-chain-id` from service to service alive. Can remove in the future though